.github/workflows/tld-update.yml

```diff
@@ -39,7 +39,18 @@ jobs:
           committer: "GitHub <noreply@github.com>"
           author: "GitHub <noreply@github.com>"
           branch: psl-gtld-update
+          labels: |
```

Comment:
Just as an FYI, in the past I had bad experience with labels with icons. And by "bad" I mean the unicode char did cause random failures in integrations.

I've personally moved away from using any icon/emoji at all in labels.